src/Sulu/Bundle/MediaBundle/Resources/js/containers/MediaSelection/tests/MediaSelection.test.js

```diff
@@ -0,0 +1,384 @@
+/* eslint-disable flowtype/require-valid-file-annotation */
+import {mount, render, shallow} from 'enzyme';
+import pretty from 'pretty';
+import React from 'react';
+import {DatagridStore} from 'sulu-admin-bundle/containers';
+import MediaSelection from '../MediaSelection';
+import MediaCardSelectionAdapter from '../../Datagrid/adapters/MediaCardSelectionAdapter';
+import MediaSelectionStore from '../stores/MediaSelectionStore';
+import CollectionStore from '../../../stores/CollectionStore';
+
+jest.mock('../stores/MediaSelectionStore', () => jest.fn());
+
+jest.mock('sulu-admin-bundle/containers', () => {
+    return {
+        Datagrid: require('sulu-admin-bundle/containers/Datagrid/Datagrid').default,
+        DatagridStore: jest.fn(function(resourceKey) {
+            const COLLECTIONS_RESOURCE_KEY = 'collections';
+
+            const collectionData = [
+                {
+                    id: 1,
+                    title: 'Title 1',
+                    objectCount: 1,
+                    description: 'Description 1',
+                },
+                {
+                    id: 2,
+                    title: 'Title 2',
+                    objectCount: 0,
+                    description: 'Description 2',
+                },
+            ];
+
+            const thumbnails = {
+                'sulu-260x': 'http://lorempixel.com/260/100',
+                'sulu-25x25': 'http://lorempixel.com/25/25',
+            };
+
+            const mediaData = [
+                {
+                    id: 1,
+                    title: 'Title 1',
+                    mimeType: 'image/png',
+                    size: 12345,
+                    url: 'http://lorempixel.com/500/500',
+                    thumbnails: thumbnails,
+                },
+                {
+                    id: 2,
+                    title: 'Title 2',
+                    mimeType: 'image/jpeg',
+                    size: 54321,
+                    url: 'http://lorempixel.com/500/500',
+                    thumbnails: thumbnails,
+                },
+            ];
+
+            this.loading = false;
+            this.pageCount = 3;
+            this.data = (resourceKey === COLLECTIONS_RESOURCE_KEY)
+                ? collectionData
+                : mediaData;
+            this.selections = [];
+            this.getPage = jest.fn().mockReturnValue(2);
+            this.getFields = jest.fn().mockReturnValue({
+                title: {},
+                description: {},
+            });
+            this.destroy = jest.fn();
+            this.sendRequest = jest.fn();
+            this.clearSelection = jest.fn();
+            this.setAppendRequestData = jest.fn();
+            this.deselectEntirePage = jest.fn();
+            this.getSchema = jest.fn().mockReturnValue({});
+        }),
+    };
+});
+
+jest.mock('sulu-admin-bundle/containers/Datagrid/registries/DatagridAdapterRegistry', () => {
+    const getAllAdaptersMock = jest.fn();
```

Comment:
This const is probably not really necessary, just inline the `jest.fn()` in the `return`.

Reply:
It is for line `85`.